release-pipeline.yml

```diff
@@ -0,0 +1,88 @@
+trigger: none
+
+resources:
+  pipelines:
+  - pipeline: microsoft_MSBuildLocator
+    source: microsoft.MSBuildLocator
+    trigger:
+      enabled: false
+
+variables:
+  SymbolsAgentPath: '$(System.DefaultWorkingDirectory)\_$(Build.DefinitionName)\Symbols'
+  SymbolsEmailContacts: 'angerlic'
+  SymbolsFeatureName: 'MSBuild'
+  SymbolsProject: 'DDE'
+  SymbolsUncPath: '\\cpvsbuild\drops\MSBuild\symbols\$(Build.DefinitionName)\$(Build.BuildNumber)\symbols.archive'
```

Comment:
uh, is this still how we do things? I guess it's ok as long as it works.

Maybe check in the conversion like this then remove this unused variable.